app/controllers/api/v1/base_controller.rb

```diff
@@ -0,0 +1,48 @@
+class Api::V1::BaseController < ApplicationController
+
+  def create
```

Comment:
Yeah I pretty much looked at the wantable2 address, creditcard, base controllers and stripped away everything i didn't need